src/test/incremental/hashes/let_expressions.rs

```diff
@@ -86,7 +86,7 @@ pub fn change_mutability_of_slot() {
 
 #[cfg(not(cfail1))]
 #[rustc_clean(cfg="cfail2",
-    except="hir_owner_nodes,typeck,optimized_mir")]
```

Comment:
any idea how this used to change the `optimized_mir` and why it doesn't do so anymore? Because the local variable is now entirely optimized out and thus there's not even a difference in mutability?

Reply:
Yep, that's exactly it!